src/lib/plugins/settings.js

```diff
@@ -7,11 +7,11 @@ function randomInt (low, high) {
 module.exports.server = function (serv, settings) {
   serv.gameMode = settings.gameMode
   serv.difficulty = settings.difficulty
-  const mcData = require('minecraft-data')(settings.version)
+  const registry = require('prismarine-registry')(settings.version)
```

Comment:
There should really be one instance of prismarine-registry, probably under server.registry, as the point of prismarine-registry is to create a copy of Minecraft data that can be added onto (with login codec data or other things). Otherwise calling methods on the registry will only update the data for that specific registry instance

Reply:
I've changed enough things already and declare that out of scope for this PR. 

Also, the registries might be using different versions because player plugins create the registry using the player protocol version.